packages/chain-helpers/src/blockchainApiConnection/BlockchainApiConnection.ts

```diff
@@ -15,40 +15,178 @@ import Blockchain from '../blockchain/Blockchain'
 let instance: Promise<Blockchain> | null
 
 export const CUSTOM_TYPES: RegistryTypes = {
+  // Runtime types
```

Comment:
Can't we use the types from the [type-definition](https://github.com/KILTprotocol/type-definitions/blob/master/src/index.ts) repo here?

Reply:
that's actually already implemented downstream, so will be merged (yeeeey, conflicts)

Reply:
or upstream ? I dunno, it's in develop

Reply:
Then perhaps you should also merge from `develop`?

Reply:
I would have done it before merge, but I can do it now if you want to test

Reply:
I think it is better just to avoid duplicating work when something in the types changes, right?